tests/e2e/data/taquito-funding-instructions.txt

```diff
@@ -1,4 +1,4 @@
-A keypair with public key hash tz3__address__ was generated for you.
+A keypair with public key hash tz__address__ was generated for you.
 To fund this account:
 1. Go to https://teztnets.xyz and click "Faucet" of the target testnet
```

Comment:
Faucets have been removed, no?

Reply:
Yep but the meaning of Faucet here is related to that website